Modules/Models/Region.swift

```diff
@@ -364,6 +375,9 @@ public enum Province: String, Codable, CaseIterable {
   case viboValentia = "VV"
   case vicenza = "VI"
   case viterbo = "VT"
+
+  // special
+  case abroad = "EX"
```

Comment:
[i] What if we use a 3 letters code here or some special characters to avoid an unrealistic collision with a new province? Does the backend support a 3 letters province in case?